src/nd.cpp

```diff
@@ -37,6 +37,9 @@
 
 using namespace godot;
 
+const double nd::INF = std::numeric_limits<double>::infinity();
+const double nd::NAN_VAL = std::nan("");
```

Comment:
Any reason why these are defined in the `cpp` file instead of the header (and be made `constexpr`)?